src/handlers/tests/account-deletion-processor-handler.test.ts

```diff
@@ -21,7 +20,20 @@ const loggerWarnSpy = jest.spyOn(logger, 'warn');
 describe('Account Deletion Processor', () => {
   let mockEvent: SQSEvent;
   let mockRecord: SQSRecord;
-  const mockContext = ContextExamples.helloworldContext;
+  const mockContext = {
```

Comment:
bit of a nitpick but would it be worth moving this object somewhere and importing it in to the test files, just like the way `mockContext` was handled before?

Reply:
Hi @liamquayle - I did this originally but then sanarcloud complained that it was not covered by tests, tried to exclude it from the test coverage but had no joy - instead could you create a low priority tech-debt ticket to tidy this up in the future.